db/migrate/20200414145010_add_section_103_message_column_to_institutions.rb

```diff
@@ -0,0 +1,5 @@
+class AddSection103MessageColumnToInstitutions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :institutions, :section_103_message, :string
```

Comment:
Does it make sense for this to have `default: 'No information available at this time'`?

Reply:
(same in the other migration)

Reply:
If so, you could remove one of the updates in the  `self.add_sec103(version_id)` method

Reply:
Definitely, sounds good!